mathml/presentation-markup/operators/mo-form-dynamic.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>&lt;mo&gt; dynamic form</title>
+    <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#tokenmo">
+    <meta name="assert" content="Verifies dynamic behavior of form.">
+    <link rel="match" href="mo-form-dynamic-ref.html">
+
+    <script>
+      function test()
+      {
+        for (var i = 1; i <= 6; i++) {
+          var row = document.getElementById("row" + i);
+          var x = document.getElementById("x" + i);
+          x.parentNode.removeChild(x);
+          row.insertBefore(x, row.firstElementChild);
+        }
+      }
+    </script>
+  </head>
+  <body onload="test()">
+
+    <!-- This test verifies that the form of the operators (and thus their spacing) is updated when you change the child list. -->
+    <!-- The test should render the same as the static reference -->
```

Comment:
Maybe the pass condition should be text in a normal html paragraph, so people can read it without checking the source.

Reply:
Done.